src/screens/TopicPageScreen/MemoizedListComponent.js

```diff
@@ -141,10 +141,7 @@ const RenderListFeed = (props) => {
 
   const topicBottomPosition = () => {
     if (hasComment) {
-      if (getTotalReaction(item) === 1) {
-        return isBlurred && getHeightReaction() + normalize(4);
-      }
-      return getHeightReaction();
+      return getHeightReaction() + normalize(4);
     }
     return getHeightFooter();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new logic in `topicBottomPosition` function seems to have removed the condition checking for `getTotalReaction(item) === 1` and `isBlurred`. If these conditions are still relevant, they should be included in the updated code. If not, ensure that this change doesn't introduce any unintended side effects.
```diff
- if (hasComment) {
-   return getHeightReaction() + normalize(4);
- }
+ if (hasComment) {
+   if (getTotalReaction(item) === 1 && isBlurred) {
+     return getHeightReaction() + normalize(4);
+   }
+   return getHeightReaction();
+ }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->